container_test.go

```diff
@@ -19,6 +19,9 @@ import (
 )
 
 func empty() IOCreation {
+	if runtime.GOOS == "windows" {
```

Comment:
:boom:

I'll have to look at it, could you add a todo with my name on it before I hit merge?